lib/command/CommandClient.js

```diff
@@ -175,7 +175,7 @@ class CommandClient extends Client {
                     if(msg.command.errorMessage) {
                         try {
                             if(typeof msg.command.errorMessage === "function") {
-                                const reply = msg.command.errorMessage();
+                                const reply = msg.command.errorMessage(msg,err);
```

Comment:
this should probably have a space between the params... not sure how the linter doesn't pick that up

Reply:
It should, yes. Additionally, you'll need to update the docs to show that an error is being passed as well.